app/services/vacancy_alert_search_builder.rb

```diff
@@ -17,4 +17,33 @@ def published_on_query
       },
     }
   end
+
+  def salary_query
```

Comment:
Shouldn't all these be private to match the access level of the methods they override?